### PR TITLE
Configure tox

### LINKED ISSUE #4: Setup the tests environment with tox

### PR DESCRIPTION
**Pull request by hugobessa**
Configures tox to test the project in multiple environments

The selected environments are: 
* {py27, py35, py36} django-110
* {py27, py35, py36} django-111
* {py35, py36} django-20

resolves #4

### CODE REVIEW

File: .python-version [hunk 1]
```diff
@@ -0,0 +1,3 @@
+2.7.13
+3.5.2
+3.6.2
```
Comment (chocoelho) [inline .python-version]: This is from your local pyenv environment. Add this to `.gitignore`

File: celerybeat_status/tests/test_views.py [hunk 2]
```diff
@@ -1,13 +1,17 @@
-from django.core.urlresolvers import reverse
 
-from core.tests.utils import (
+from celerybeat_status.tests.utils import (
     SuperuserBaseTestCase, TestRequiresSuperuser)
 
+try:
+    from django.core.urlresolvers import reverse
+except ImportError:
```
Comment (chocoelho) [inline celerybeat_status/tests/test_views.py]: I think that checking by version is more elegant and more readable. Check this:
```python
from django.utils import version as django_version

if django_version.get_complete_version() < (2, 0, 0):
    from django.core.urlresolvers import reverse
else:
    from django.urls import reverse
```

**Review by chocoelho — CHANGES_REQUESTED** [review]
Please add local pyenv configuration to `.gitignore` and make use of version to check on from where to import a module instead of using exceptions. The latter is preferable since is way more readable on why you're trying the alternate import.

**Comment by hugobessa** [comment]
Thank you for the review @chocoelho! I've fixed the points you mentioned

**Review by chocoelho — APPROVED** [review]

**Pull request merged**